### PR TITLE
[DONOTMERGE] [AIFBR-7715] Add Language Version Metadata

### PR DESCRIPTION
**Pull request by adeepsri**
Caters: [AIFBR-7715] Add Language Version Metadata

[AIFBR-7715]: https://uipath.atlassian.net/browse/AIFBR-7715

### CODE REVIEW

File: language_version_metadata/Python27__0__metadata.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[{
```
Comment (vivekjhauipath) [inline language_version_metadata/Python27__0__metadata.json:1]: we only support CPU serving for these?

File: language_version_metadata/Python3X__0__metadata.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[{
```
Comment (vivekjhauipath) [inline language_version_metadata/Python3X__0__metadata.json:1]: shouldn't we default to python 36 here and have all 4

File: language_version_metadata/Python27__0__metadata.json (hunk 1 above)
Comment (adeepsri) [reply language_version_metadata/Python27__0__metadata.json:1]: yes

File: language_version_metadata/Python3X__0__metadata.json (hunk 2 above)
Comment (adeepsri) [reply language_version_metadata/Python3X__0__metadata.json:1]: these are here just for backward compatibility.

**Review by vivekjhauipath — APPROVED** [review]
LGTM. Though not needed as they are part of changelog

**Review by mohitseth-nitrr — APPROVED** [review]
LGTM

**Review by rajivml — APPROVED** [review]

**Comment by adeepsri** [comment]
This PR is not required as currently all the languages are available in db changelog

**Comment by adeepsri** [comment]
Closing this PR as it is not required.

**Pull request closed without merging**